src/Uno.Foundation.Runtime.WebAssembly/Interop/Runtime.wasm.cs

```diff
@@ -55,12 +82,14 @@ public static class WebAssemblyRuntime
 
 		private static readonly Lazy<ILogger> _logger = new Lazy<ILogger>(() => typeof(WebAssemblyRuntime).Log());
 
-		
 		public static bool IsWebAssembly { get; }
 			// Origin of the value : https://github.com/mono/mono/blob/a65055dbdf280004c56036a5d6dde6bec9e42436/mcs/class/corlib/System.Runtime.InteropServices.RuntimeInformation/RuntimeInformation.cs#L115
 			= RuntimeInformation.IsOSPlatform(OSPlatform.Create("WEBASSEMBLY")) // Legacy Value (Bootstrapper 1.2.0-dev.29 or earlier).
 			|| RuntimeInformation.IsOSPlatform(OSPlatform.Create("BROWSER"));
 
+		internal static bool IsNetCore { get; }
+			= RuntimeInformation.FrameworkDescription.StartsWith(".NET")
+			&& !RuntimeInformation.FrameworkDescription.StartsWith(".NET Framework");
```

Comment:
Nothing stronger than this weak detection mecanism?

Reply:
Yes, testing for the presence of `AssemblyLoadContext` works better:

```
internal static bool IsNetCore { get; } = Type.GetType("System.Runtime.Loader.AssemblyLoadContext") != null;
```

Inlining has to be disabled on the caller site for this to work.